src/lex/cursor.rs

```diff
@@ -92,7 +98,7 @@ impl<'a> Cursor<'a> {
         self.col
     }
 
-    pub(crate) fn err_expected_token(&self, message: &'static str) -> LexError {
+    pub(crate) fn generate_err_expected_token(&self, message: &'static str) -> LexError {
```

Comment:
Thanks for contribution. It looks almost all good to me. But I'd like to make `generate_err_expected_token` more shorter like `make_err_expected_token`, because it is used frequently.